demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -93,13 +93,17 @@ def upload_zip(
     client: demisto_client,
     skip_validations: bool,
     target_demisto_version: Version,
+    marketplace: MarketplaceVersions,
 ) -> bool:
     """
     Used to upload an existing zip file
     """
     if path.suffix != ".zip":
         raise RuntimeError(f"cannot upload {path} as zip")
-    if target_demisto_version < MINIMAL_UPLOAD_SUPPORTED_VERSION:
+    if (
+        marketplace == MarketplaceVersions.XSOAR
+        and target_demisto_version < MINIMAL_UPLOAD_SUPPORTED_VERSION
```

Comment:
why not `<=` ?

Reply:
If `target_demisto_version == MINIMAL_UPLOAD_SUPPORTED_VERSION` and we on XSOAR, we *do* want to upload.
The only reason to raise this error is if we are on XSOAR which is below (strictly) the minimum upload supported version